app/models/game.rb

```diff
@@ -1,5 +1,6 @@
 class Game < ActiveRecord::Base
-	belongs_to :user
+	belongs_to :white_player, class_name: "User"
+	belongs_to :black_player, class_name: "User"
 	has_many :pieces
```

Comment:
Good call.  However, I think we had decided on a has_many :users when we developed the spreadsheet and that might be a better way to go.